script/hassfest/translations.py

```diff
@@ -131,11 +131,13 @@ def translation_value_validator(value: Any) -> str:
     - prevents strings with single quoted placeholders
     - prevents combined translations
     """
-    value = cv.string_with_no_html(value)
-    value = string_no_single_quoted_placeholders(value)
-    if RE_COMBINED_REFERENCE.search(value):
+    string_value = cv.string_with_no_html(value)
+    string_value = string_no_single_quoted_placeholders(string_value)
+    if RE_COMBINED_REFERENCE.search(string_value):
         raise vol.Invalid("the string should not contain combined translations")
-    return str(value)
+    if string_value != string_value.strip(" "):
```

Comment:
`str.strip` by default removes whitespace.

https://docs.python.org/3/library/stdtypes.html#str.strip

Reply:
Some integrations have trailing `\n` which seemed intentional
Those would be stripped also with default  strip.

Reply:
Please give an example.

Reply:
> Please give an example.

See #126446 for where the `\n` seems intentional